Alloy/commands/purgetss/index.js

```diff
@@ -0,0 +1,50 @@
+const { exec } = require('child_process');
+const commandExistsSync = require('command-exists').sync;
```

Comment:
`commandExistsSync` is unused

Reply:
I forgot to remove `command-exists` didn't work as expected!!